CrewChiefV4/UserInterface/MainWindow.cs

```diff
@@ -33,22 +33,23 @@ public partial class MainWindow : Form
 
         private String baseDriverNamesDownloadLocation;
         private String updateDriverNamesDownloadLocation;
+        private String update2DriverNamesDownloadLocation;
         private String driverNamesTempFileName = "temp_driver_names.zip";
-        private Boolean getBaseDriverNames = false;
+        private String drivernamesToDownload;
 
         private String baseSoundPackDownloadLocation;
         private String updateSoundPackDownloadLocation;
         private String update2SoundPackDownloadLocation;
+        private String update3SoundPackDownloadLocation;
         private String soundPackTempFileName = "temp_sound_pack.zip";
-        private Boolean getBaseSoundPack = false;
-        private Boolean getFirstUpdateSoundPack = false;
+        private String soundPackToDownload;
 
         private String basePersonalisationsDownloadLocation;
         private String updatePersonalisationsDownloadLocation;
         private String update2PersonalisationsDownloadLocation;
+        private String update3PersonalisationsDownloadLocation;
```

Comment:
those variables could also be in a List just as versions are.

Reply:
Yeah, will refactor this crap

Reply:
exactly... so that ideally it is new entry in List, and done

Reply:
I am just a bit worried about risks, as we are testing on live customers.

Reply:
reworked it all - it a bit neater now and easier to add new packs